packages/commonwealth/client/scripts/state/api/communities/editCommunityTags.ts

```diff
@@ -31,9 +29,14 @@ const editCommunityTags = async ({
 const useEditCommunityTagsMutation = () => {
   return useMutation({
     mutationFn: editCommunityTags,
-    onSuccess: async ({ tags, community_id }) => {
-      // @ts-expect-error StrictNullChecks
-      app.config.chainCategoryMap[community_id] = tags;
+    onSuccess: ({ CommunityTags = [], community_id }) => {
+      const community = app.config.chains.getById(community_id);
+      if (community) community.updateTags(CommunityTags);
+
+      // reset active communities cache
+      const key = [ApiEndpoints.FETCH_ACTIVE_COMMUNITIES];
+      queryClient.cancelQueries(key).catch(console.error);
+      queryClient.refetchQueries(key).catch(console.error);
```

Comment:
just want to confirm that this means that if I edit community tag, then the whole list of 300 communities have to be refetched? If that's the case, maybe we should just modify the cache with those tags?

Reply:
done